src/sections/Community/Calendar/index.js

```diff
@@ -129,6 +132,28 @@ const Calendar = () => {
         </Container>
         <Table columns={columns} data={data} />
       </div>
+      <br/>
+      <Container>
```

Comment:
Why this section is added?

Reply:
I have deleted that file, It wasn't a part of this PR.